cmd/lnmuxd/config.go

```diff
@@ -19,6 +19,9 @@ type Config struct {
 
 	// IdentityKey is the private key that is used to sign invoices.
 	IdentityKey string `yaml:"identityKey"`
+
+	// AutoSettle indicates that payments should be accepted automatically.
+	AutoSettle bool `yaml:"autoSettle"`
```

Comment:
I wonder if you could keep 1 bit in the payment addr to indicate if auto settling mode

Reply:
agreed